pkgs/build-support/setup-hooks/make-wrapper.sh

```diff
@@ -1,8 +1,28 @@
+# construct an executable file that wraps the actual executable
```

Comment:
Yes, you can. But why would you? If you keep it out, it is also much easier to include it in other documentation, like the manual.